src/request-pipeline/utils.ts

```diff
@@ -17,6 +17,13 @@ import { PassThrough } from 'stream';
 import { getText, MESSAGE } from '../messages';
 import logger from '../utils/logger';
 
+const NODE_MAX_HEADER_SIZE = 81920;
```

Comment:
Why does we use this value? Please, add a link to specification, comment or something else.

Reply:
Calculate all sizes in bytes.

Reply:
This limitation turned out to be unnecessary. Node did previously limit header sizes to 80 Kb but gave no apparent reason as to why.